pkg/operator/ceph/cluster/mgr/mgr_test.go

```diff
@@ -181,116 +177,52 @@ func TestUpdateServiceSelectors(t *testing.T) {
 	}
 	c := &Cluster{spec: spec, context: ctx, clusterInfo: clusterInfo}
 
-	t.Run("initial active daemon", func(t *testing.T) {
-		activeDaemon := "a"
-		err := c.reconcileServices(activeDaemon)
+	// Make sure we remove the daemon_id label from the selector
+	// of all services with a label "app=rook-ceph-mgr"
+	t.Run("remove daemon_id from mgr services", func(t *testing.T) {
+		svc_1 := corev1.Service{
```

Comment:
nit: don't use underscores in golang variable names
```suggestion
		svc1 := corev1.Service{
```

Reply:
fixed